app/uk/gov/hmrc/thirdpartyapplication/models/Application.scala

```diff
@@ -174,12 +175,16 @@ object ApplicationResponse {
     case a: Standard => a.redirectUris
     case _ => List.empty
   }
+
   def termsAndConditionsUrl(data: ApplicationData): Option[String] = data.access match {
```

Comment:
Are these used anywhere? in this project?

Reply:
Yes, they are used in ApplicationResponse.apply - could be made private though?